docs/docs/reference/crd-reference/metric.md

```diff
@@ -37,49 +37,51 @@ spec:
 ```
 
 ## Fields
+<!-- markdownlint-disable MD007 -->
```

Comment:
you always need re-enable the rule  at the end of the section you want to ignore it for

Reply:
fixed